samples/contrib/aws-samples/ground_truth_pipeline_demo/README.md

```diff
@@ -18,18 +18,21 @@ Create a s3 bucket and run [this python script](https://github.com/kubeflow/pipe
 
 ## Amazon Cognito user groups
 
-To use the workteam component, you must have an Amazon Cognito user group(s) set up with the workers you want for the workteam.
+From Cognito note down Pool ID, User Group Name and client ID  
+You need this information to fill arguments user_pool, user_groups and client_ID
 
-For this demo, please create a user group with only yourself in it.
+[Official doc for Amazon Cognito](https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-getting-started.html)
 
-If this is your first time creating a private workteam in the region you are using, follow the steps below to set up a new user pool, user group, and app client ID on Amazon Cognito.
-If you already have private workteams in the region you are using, skip step 1 and do steps 2 and 3 in the user pool that is already being used.
+For this demo you can create a new user pool (if you don't have one already).   
 
-Steps:
-1. Create a user pool in Amazon Cognito. Configure the user pool as needed, and make sure to create an app client. The Pool ID will be found under General settings.
-2. After creating the user pool, go to the Users and Groups section and create a group. Create users for the team, and add those users to the group.
-3. Under App integration > Domain name, create an Amazon Cognito domain for the user pool.
+AWS console -> Amazon SageMaker -> Ground Truth, Labeling workforces -> Private -> Create Private Team -> Give it "KFP-ground-truth-demo-pool" name and use your email address -> Create Private team -> Click on the radio button and from summary note down the "Amazon Cognito user pool", "App client" and "Labeling portal sign-in URL" -> click on the team name that you created and note down "Amazon Cognito user group"
```

Comment:
Can we make this better by using a blog or aws docs as reference for this process? 

If we don't have an existing reference, can you do something like [this link](https://docs.aws.amazon.com/elastic-inference/latest/developerguide/setting-up-ei.html#eia-launch) which guides user on console process to launch an instance with EI